### PR TITLE
rename to upsint

### LINKED ISSUE #31: rename to upsint
upstream-integration

### PR DESCRIPTION
**Pull request by TomasTomecek**
Fixes #31
+ put short license on top of each source file
+ use setuptools-scm

### CODE REVIEW

File: setup.cfg [hunk 1]
```diff
@@ -1,7 +1,9 @@
+[bdist_wheel]
+universal = 1
+
 [metadata]
-name = tool
-version = attr: tool.version.__version__
-url = https://github.com/user-cont/tool
+name = upsint
+url = https://github.com/user-cont/upsint
```
Comment (lachmanfrantisek) [inline setup.cfg]: What about packit org, isn't it a better place for this?

Comment (TomasTomecek) [reply setup.cfg]: I don't care personally. We can vote if you want.

Comment (dhodovsk) [reply setup.cfg]: I vote for packit! It can be taken as just another piece we develop to ease the developer's life.

**Comment by TomasTomecek** [comment]
here we are! upsint, welcome to packit-service

**Pull request merged**